crd-docs/crd/kiali.io_kialis.yaml

```diff
@@ -1490,6 +1490,9 @@ spec:
                       enabled:
                         description: "When 'true', the profiler will be enabled and accessible at /debug/pprof/ on the Kiali endpoint."
                         type: boolean
+                  require_auth:
+                    description: "When true, the /api endpoint will require users to authenticate themselves. When false, users need not authenticate with Kiali in order to get basic runtime info about the server via the /api endpoint. This setting is ignored if auth.strategy is 'anonymous'."
```

Comment:
```suggestion
                    description: "When true, the /api endpoint (and all other endpoints), will require users to authenticate themselves. When false, users need not authenticate with Kiali in order to get basic runtime info about the server via the /api endpoint. This setting is ignored if auth.strategy is 'anonymous'."
```

Reply:
That is not true. NOT all other endpoints... specifically, there are /api/auth/XX endpoints that still are (and are required to me) unauthorized.

Right now this fix is very specific and only affects that single `api/` endpoint

Reply:
hmmm, OK.  Then maybe I don't love the setting name.  But not worth changing.

Reply:
I thought about the  name of the setting, but I was expecting something in the future where that setting might also affect (like maybe we add some other "version" or "status" time endpoints that we also want unauth'ed by default but allow a user to put it behind auth. So I tried to name it more generic as opposed to something like "make_api_endpoint_secure" or something like that.